### ISSUE TITLE
Trying to set up gcr triggers.

### ISSUE THREAD

**Issue by duxbuse** [google-terraform-modules/terraform-google-cloudbuild-gcr#1]
Using this module I was able to easily create a source repo as well as the trigger, is it possible to extend the functionality to also initialise the new source repo to mirror an existing github or bitbucket repo?

**Comment by perriea** [comment]
Just to be sure you talk about `Source Repositories` ?
At the very beginning I did that, but I deleted this feature because I thought that the creation of the mirror repo was too much.

I think I will add it during a module update :)

**Comment by duxbuse** [comment]
Yes I already have code on github that I want access to on google cloud. So initialising a google source repository that mirrors my github would be very helpful.

**Comment by cilindrox** [comment]
This would come in handy too. Was looking at the google tf provider as well as the [docs for mirroring](https://cloud.google.com/source-repositories/docs/mirroring-a-github-repository) and couldn't think of a way to enabling this other than through a [`MirrorConfig`](https://cloud.google.com/source-repositories/docs/reference/rest/v1/projects.repos#MirrorConfig) api call.
